### DISCUSSION TITLE
Issue with NGINX - Error Code 1

### DISCUSSION THREAD

**Discussion by rarodrigo** [netbox-community/netbox-docker#381] [Getting Help]
Hello Team,

This issue is same related with [Docker Issue](https://github.com/netbox-community/netbox-docker/issues/22).

For me situation the logs are:
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration,
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/,
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh,
10-listen-on-ipv6-by-default.sh: Getting the checksum of /etc/nginx/conf.d/default.conf,
10-listen-on-ipv6-by-default.sh: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf,
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh,
/docker-entrypoint.sh: Configuration complete; ready for start up,
2020/12/14 13:58:05 [emerg] 1#1: host not found in upstream "netbox" in /etc/netbox-nginx/nginx.conf:29,
nginx: [emerg] host not found in upstream "netbox" in /etc/netbox-nginx/nginx.conf:29,

Docker Compose
docker-compose version 1.25.5, build unknown

Is there any another recommendation about it?

**Comment by cimnine** [comment]
It seems you have not started Netbox. What does `docker-compose ps` print? What do you see in `docker-compose logs netbox`?

**Comment by rarodrigo** [comment]
Followed status about your mention.

**Logs:**

**r:~/netbox-docker$** docker-compose ps
            Name                           Command               State     Ports  

netbox-docker_netbox-worker_1   python3 /opt/netbox/netbox ...   Exit 1           
netbox-docker_netbox_1          /opt/netbox/docker-entrypo ...   Exit 1           
netbox-docker_nginx_1           /docker-entrypoint.sh ngin ...   Exit 1           
netbox-docker_postgres_1        docker-entrypoint.sh postgres    Up       5432/tcp
netbox-docker_redis-cache_1     docker-entrypoint.sh sh -c ...   Up       6379/tcp
netbox-docker_redis_1           docker-entrypoint.sh sh -c ...   Up       6379/tcp

**Logs:**

**o:~/netbox-docker$** docker-compose logs netbox
Attaching to netbox-docker_netbox_1
netbox_1         | standard_init_linux.go:219: exec user process caused: exec format error

In this case the Netbox start, however after receive message by 3 times docker is confirmed in "Stopped" status.

**Reply by cimnine** [reply]
```
 netbox_1 | standard_init_linux.go:219: exec user process caused: exec format error
```

Netbox does not start. Something seems wrong on your end. How did you download netbox-docker? Have you changed any of the files?
